Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/RenamingPolicy.cs

```diff
@@ -214,13 +214,13 @@ private static bool IsFileMask(string input)
             var b = false;
             if (input == null)
                 return false;
-            
+
             if (input.IndexOf("*", StringComparison.InvariantCulture) >= 0)
                 b = true;
-            
+
             if (input.IndexOf("?", StringComparison.InvariantCulture) >= 0)
                 b = true;
-            
+
             return b;
         }
 
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/8c4346a7a0aded599281b906d6700bd7508cc79c/Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/RenamingPolicy.cs#L227
This method can be `static`.

https://github.com/FrendsPlatform/Frends.FTP/blob/8c4346a7a0aded599281b906d6700bd7508cc79c/Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/RenamingPolicy.cs#L236
This method can be `static`.

https://github.com/FrendsPlatform/Frends.FTP/blob/8c4346a7a0aded599281b906d6700bd7508cc79c/Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/RenamingPolicy.cs#L269
This method can be `static`.